src/pages/events/commands.md

```diff
@@ -105,9 +105,11 @@ The command supports observer events by default. You must perform additional ste
 
 You can also subscribe to a plugin event if it was registered in the `app/etc/config.php` file and subsequently unsubscribed with the [`events:unsubscribe` command](#unsubscribe-from-a-commerce-event). [Register events](./module-development.md#register-events) describes the format of these files.)
 
+You can also create and subscribe to a rule event. Rule events allow you to determine the conditions that the event service uses to emit a native or custom events to your application. See [Create rule events](./rule-events.md) for detailed information and examples.
```

Comment:
'event service' could be replaced with something like the 'Commerce events client module'